modules/apps/27-interchain-accounts/host/types/params.go

```diff
@@ -57,3 +65,18 @@ func validateEnabled(i interface{}) error {
 
 	return nil
 }
+
+func validateAllowlist(i interface{}) error {
+	allowMsgs, ok := i.([]string)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	for _, typeURL := range allowMsgs {
+		if typeURL == "" {
```

Comment:
```suggestion
		if strings.TrimSpace(typeURL) == "" {
```
you'll need to add strings import as well

Reply:
Ah good spot! Lemme fix this :)